plugins/Jobs.cpp

```diff
@@ -160,32 +160,50 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
     } 
     
     // ----------------------------------------------------------------------
-    else if (SIEquals(request.methodLine, "CreateJob")) {
-        // If unique flag was passed and the job exist in the DB, then we can
-        // finish the command without escalating to master.
-        if (!request.test("unique")) {
-            // Not unique; need to process
-            return false;
+    else if (SIEquals(request.methodLine, "CreateJob") || SIEquals(request.methodLine, "CreateJobs")) {
+        vector<STable> jsonJobs;
```

Comment:
Because when I created it, I thought I was going to need to access the elements in the vector and I cannot do that with lists. Anyways, I checked and there shouldn't be any difference to choose one over the other one. Do you want me to change it?